Sming/SmingCore/Network/HttpServer.h

```diff
@@ -89,11 +97,6 @@ class HttpServer : public TcpServer
 protected:
 	TcpConnection* createClient(tcp_pcb* clientTcp) override;
 
-protected:
-#ifdef ENABLE_SSL
-	int minHeapSize = 16384;
```

Comment:
@mikee47 What are the advantages of setting the `minHeapSize` value for every constructor instead of the old way as initial value for the protected variable `minHeapSize` ? What is the issue that this change is addressing?

Reply:
This declaration creates a new `int minHeapSize` which hides the one in `TcpServer`. A simpler, probably better, way would be to modify the statement in `TcpServer.h` instead of putting it here. That would change behaviour, but would it be an issue?

Reply:
Ok, I see that this change https://github.com/SmingHub/Sming/commit/fa422059039a901f9cb23786e60b2cffbfac0fb9 changed `int minHeapSize` to `size_t minHeapSize` in TcpServer thus making the protected int minHeapSize in HttpServer to be a completely new one.

> A simpler, probably better, way would be to modify the statement in `TcpServer.h` instead of putting it here.

Can you elaborate a bit?

Reply:
In `TcpServer.h`:

```
protected:
#ifdef ENABLE_SSL
	size_t minHeapSize = 16384;
#else
	size_t minHeapSize = 3000;
#endif
```

Reply:
```
protected:
#ifdef ENABLE_SSL
	size_t minHeapSize = 16384;
#else
	size_t minHeapSize = 3000;
#endif
```

Ok, understood. Let's have that change then.